### ISSUE TITLE
sum-of-multiples: update tests to version 1.1.0

### ISSUE THREAD

**Issue by N-Parsons** [exercism/python#1022]
As identified in #784, some of the exercise test-suites are out-of-date, `sum-of-multiples` is one of them. It would therefore be really helpful for people to go through and update the test suite to match the canonical data.

Tests should typically be ordered according to their order in the canonical data, since this makes it easier to check and update them in future. If you think that there's a better order, please let us know your reasoning in your PR.

If there are some additional tests that currently exist in this track but which aren't in the canonical data, these can be kept if you think that they are useful. They should be placed at the bottom of the test cases, and it would also be super-helpful if you could add a comment before them saying `# Additional tests for this track` to make it clear that they aren't part of the canonical data.

The version string should be placed just before the first test class and followed by a blank line. For this exercise, the version string is:
```
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
```

**Test suite:** [exercism/python/exercises/sum-of-multiples/sum_of_multiples_test.py](https://github.com/exercism/python/blob/master/exercises/sum-of-multiples/sum_of_multiples_test.py)
**Canonical data:** [exercism/problem-specifications/exercises/sum-of-multiples/canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/sum-of-multiples/canonical-data.json)

**Comment by kishan3** [comment]
Sir I can see version is already 1.1.0 in /sum-of-multiples/canonical-data.json . I am not sure what we have to do for the issue?

**Comment by kishan3** [comment]
Ah I too added #1028

**Comment by N-Parsons** [comment]
Hi @kishan3, the issue requires updating of the test data within this track's exercise test-suite (`{exercise_name}_test.py`) to match the data within the canonical data. Any missing tests (those in the canonical data but not our test-suite) need to be implemented.

**Comment by kishan3** [comment]
Thank you sir for the explaination. Will update other PRs as this is fixed in #1026 it seems.
